.github/workflows/move_to_in_review.yml

```diff
@@ -0,0 +1,39 @@
+name: Move PR to In Review
+on:
+  pull_request_target:
+    types:
+      - review_requested
+
+env:
+  PROJECT_ID: "PVT_kwDOAKRRkc4AfZil"
+  STATUS_FIELD_ID: "PVTSSF_lADOAKRRkc4AfZilzgUwVMs"
+  IN_REVIEW_ID: "618f8b62"
```

Comment:
It would be good to limit the number of times these are hardcoded in the codebase.  Is it possible to use your existing Python scripts to eliminate the need to hardcode these here?